blackonnx/examples/tutorial.md

```diff
@@ -0,0 +1,93 @@
+# Classification tutorial
+
+The two sample models in `examples/models` were trained on a simple flower classification task, using data available [here](https://public.roboflow.com/classification/flowers_classification/2). (license: Public Domain)
+
+## Preparation
+
+You can use the Docker image from `dockerfiles` folder of this repository to go through the examples as it contains all necessary packages.
+
+Then, we need to install the package of this repository with the following command:
+
+```bash
+user$ pip3 install blackonnx
+```
```

Comment:
As the library is available on PyPI, this manual step can be removed, by adding blackonnx to the docker image?

Reply:
Yes, we will remove them. Thx!

Reply:
db2d940